dropbox/dropbox.py

```diff
@@ -479,6 +521,28 @@ def _save_body_to_file(self, download_path, http_resp, chunksize=2**16):
                 for c in http_resp.iter_content(chunksize):
                     f.write(c)
 
+    def with_path_root(self, path_root):
+        """
+        Creates a clone of the Dropbox instance with the Dropbox-API-Path-Root header
+        as the appropriate serialized instance of PathRoot.
+
+        For more information, see
+        https://www.dropbox.com/developers/reference/namespace-guide#pathrootmodes
```

Comment:
I missed this before, but you might consider using [link markup](http://www.sphinx-doc.org/en/stable/rest.html#external-links) to render this as a hyperlink in generated docs.

Reply:
```
Use `Link text <http://example.com/>`_ for inline web links. If the link text should be the web address, you don’t need special markup at all, the parser finds links and mail addresses in ordinary text.
```

No need for markup, the html should automatically link

Reply:
Ooh! I learned something today! 😊